### ISSUE TITLE
Call to Action Narrow - change font color on oranges for accessibility

### ISSUE THREAD

**Issue by mwllr** [ATLAS-Illinois/Illinois-Framework-Theme#861]
Call to Action Narrow
On orange and orange gradient, we can't use white for the body. 
Orange: body-illini blue, link-illini blue with underline, hover/focus-illini blue with illini-blue dashed outline.
Orange Gradient: body-black, link-black with underline, hover/focus-black with black dashed outline.

![Screen Shot 2022-10-24 at 2 22 18 PM](https://user-images.githubusercontent.com/54747669/197608606-5282a916-a8ec-4054-996b-bf73d69eb803.png)
![Screen Shot 2022-10-24 at 2 22 57 PM](https://user-images.githubusercontent.com/54747669/197608607-994b50a1-abb1-4174-94a4-d7a1d1528e2d.png)

**Comment by mwllr** [comment]
@magjarvis - the orange gradient is an issue, so we will need to make the orange gradient exactly like the orange.

**Comment by magjarvis** [comment]
@mwllr so, will text on orange gradient need to be il-blue or black?

**Comment by mwllr** [comment]
@magjarvis - changed it to blue. Looks great! approved. Thanks.

![Image](https://user-images.githubusercontent.com/54747669/219090442-c88394a8-e2f2-40d8-b0ae-3245f5273297.png)
